features/multiple_hookfiles.feature

```diff
@@ -47,7 +47,7 @@ Feature: Multiple hookfiles with a glob
           print("It's me, File3")
 
       """
-    When I run `dredd ./apiary.apib http://localhost:4567 --server "ruby server.rb" --language "dredd-hooks-python" --hookfiles ./hookfile1.py --hookfiles ./hookfile2.py --hookfiles ./hookfile_*.py`
+    When I run `dredd ./apiary.apib http://localhost:4567 --server="ruby server.rb" --language="dredd-hooks-python" --hookfiles=./hookfile1.py --hookfiles=./hookfile2.v --hookfiles=./hookfile_*.py`
```

Comment:
Hi @honzajavorek, this looks like a misspelling: `--hookfiles=./hookfile2.v` should be `--hookfiles=./hookfile2.py`, which I think explains the [first error](https://travis-ci.org/apiaryio/dredd-hooks-python/jobs/194513087#L613). (But probably not the second.)